Doc/tools/templates/download.html

```diff
@@ -18,23 +18,23 @@
 <table class="docutils">
   <tr><th>Format</th><th>Packed as .zip</th><th>Packed as .tar.bz2</th></tr>
   <tr><td>PDF (US-Letter paper size)</td>
-    <td><a href="{{ dlbase }}/python-{{ release }}-docs-pdf-letter.zip">Download</a> (ca. 13 MB)</td>
-    <td><a href="{{ dlbase }}/python-{{ release }}-docs-pdf-letter.tar.bz2">Download</a> (ca. 13 MB)</td>
+    <td><a href="{{ dlbase }}/python-{{ release }}-docs-pdf-letter.zip">Download</a> (ca. 13 MiB)</td>
+    <td><a href="{{ dlbase }}/python-{{ release }}-docs-pdf-letter.tar.bz2">Download</a> (ca. 13 MiB)</td>
   </tr>
   <tr><td>PDF (A4 paper size)</td>
-    <td><a href="{{ dlbase }}/python-{{ release }}-docs-pdf-a4.zip">Download</a> (ca. 13 MB)</td>
-    <td><a href="{{ dlbase }}/python-{{ release }}-docs-pdf-a4.tar.bz2">Download</a> (ca. 13 MB)</td>
+    <td><a href="{{ dlbase }}/python-{{ release }}-docs-pdf-a4.zip">Download</a> (ca. 13 MiB)</td>
+    <td><a href="{{ dlbase }}/python-{{ release }}-docs-pdf-a4.tar.bz2">Download</a> (ca. 13 MiB)</td>
   </tr>
   <tr><td>HTML</td>
-    <td><a href="{{ dlbase }}/python-{{ release }}-docs-html.zip">Download</a> (ca. 9 MB)</td>
-    <td><a href="{{ dlbase }}/python-{{ release }}-docs-html.tar.bz2">Download</a> (ca. 6 MB)</td>
+    <td><a href="{{ dlbase }}/python-{{ release }}-docs-html.zip">Download</a> (ca. 9 MiB)</td>
+    <td><a href="{{ dlbase }}/python-{{ release }}-docs-html.tar.bz2">Download</a> (ca. 6 MiB)</td>
   </tr>
   <tr><td>Plain Text</td>
-    <td><a href="{{ dlbase }}/python-{{ release }}-docs-text.zip">Download</a> (ca. 3 MB)</td>
-    <td><a href="{{ dlbase }}/python-{{ release }}-docs-text.tar.bz2">Download</a> (ca. 2 MB)</td>
+    <td><a href="{{ dlbase }}/python-{{ release }}-docs-text.zip">Download</a> (ca. 3 MiB)</td>
+    <td><a href="{{ dlbase }}/python-{{ release }}-docs-text.tar.bz2">Download</a> (ca. 2 MiB)</td>
   </tr>
  <tr><td>EPUB</td>
-    <td><a href="{{ dlbase }}/python-{{ release }}-docs.epub">Download</a> (ca. 5.5 MB)</td>
+    <td><a href="{{ dlbase }}/python-{{ release }}-docs.epub">Download</a> (ca. 5.5 MiB)</td>
```

Comment:
Are you sure the size in MB and MiB is the same? Are binary prefixes preferenced here?

Reply:
I'm quite sure that the file size were compute with an unit of 1024 * 1024 bytes, MiB.

Reply:
The size of epub-file in 3.7 is 5.426 MB or 5.175 MiB.  5.5 is closer to 5.426, but in any case this is an approximate estimation. The question is what unit is more human-readable. What units are used by a file manager in Windows and OS X?

Reply:
> What units are used by a file manager in Windows and OS X?

https://blogs.msdn.microsoft.com/oldnewthing/20090611-00/?p=17933

"Explorer is just following existing practice. Everybody (to within experimental error) refers to 1024 bytes as a kilobyte, not a kibibyte."

I prefer KiB rather than KB since KiB is defined as 1024 bytes, whereas KB (or kB) may be 1000 or 1024 bytes.

> The size of epub-file in 3.7 is 5.426 MB or 5.175 MiB. 5.5 is closer to 5.426, but in any case this is an approximate estimation. 

*I* don't think that it's interesting to give the file size. If we want to keep it, maybe we should drop digits after the dot.

These numbers are likely outdated, it's very hard to keep them up to date.

Reply:
If Windows Explorer uses the multiplier 1024 but call it KB, it is good to use binary prefixes here. If it uses the multiplier 1000, perhaps it is better to use decimal prefixes.

Reply:
> If it uses the multiplier 1000, perhaps it is better to use decimal prefixes.

I'm not aware of any application using decimal prefixes...

Reply:
The volume of hard disks usually is specified with decimal prefixes.

I just have tested, Windows uses binary multipliers (but decimal prefixes), thus it is better to use binary multipliers for getting the same numbers, but with correct binary prefixes.

Reply:
Hard drive manufacturers use decimal prefixes, all software I know of uses binary. That's most of the cause behind a 3 TB drive appearing as 2.7 GB to Windows/Linux.

That said, unless you actually care about byte-precision, 5.5 MB is mentally equivalent to 5.5 MiB, or 6 MB, or 5 MB.